msteams-platform/resources/moodleInstructions.md

```diff
@@ -83,157 +83,169 @@ Next, you'll need to register Moodle as an application in your Azure AD. We've p
 
-1. Once the page refreshes you should see a new section **Choose connection method**. Select the checkbox labeled **Default** and then select **Save changes** again.
+1. Once the page refreshes, you can see a new section **Choose connection method**. Select the checkbox labeled **Default** and then select **Save changes** again.
 
-1. Once the page refreshes you will see another new section **Admin consent & additional information**.
-    * Select the **Provide Admin Consent** link, enter your Microsoft 365 Global Administrator credentials, then **Accept** to grant the permissions.
-    * Next to the **Azure AD Tenant** field select the **Detect** button.
-    * Next to the **OneDrive for Business URL** select the **Detect** button.
+1. Once the page refreshes you can see another new section **Admin consent & additional information**.
+    * Select **Provide Admin Consent** link, enter your Microsoft 365 Global Administrator credentials, then **Accept** to grant the permissions.
+    * Next to the **Azure AD Tenant** field, select the **Detect** button.
+    * Next to the **OneDrive for Business URL**, select the **Detect** button.
     * Once the fields populate, select the **Save changes** button again.
 
 1. Select the **Update** button to verify the installation, then **Save changes**.
 
-1. Next, you'll need to synchronize users between your Moodle server and Azure AD. Depending on your environment, you may select different options during this stage. To get started:
+1. Next, you must synchronize users between your Moodle server and Azure AD. Depending on your environment, you can select different options during this stage. To get started:
     * Switch to the **Sync Settings tab**
-    * In the **Sync users with Azure AD** section, select the checkboxes that apply to your environment. Typically, you would, at a minimum, select the following:  
+    * In the **Sync users with Azure AD** section, select the checkboxes that apply to your environment. You must select the following:  
 
         ✔ Create accounts in Moodle for users in Azure AD . 
         ✔ Update all accounts in Moodle for users in Azure AD.  
 
-    * In the **User Creation Restriction** section, you can setup a filter to limit the Azure AD users that will be synced to Moodle.
-    * The **User Field Mapping** section will allow you to customize the Azure AD to Moodle User Profile field mapping.
-    * In the **Teams Sync** section you can choose to automatically create Groups (i.e. teams) for some, or all, of your existing Moodle courses.
+    * In the **User Creation Restriction** section, you can setup a filter to limit the Azure AD users that is synced to Moodle.
+    * The **User Field Mapping** section allows you to customize the Azure AD to Moodle User Profile field mapping.
+    * In the **Teams Sync** section, you can choose to automatically create Groups, such as teams for some, or all, of your existing Moodle courses.
```

Comment:
Please check for all instances of choose. MSTP recommends Select instead of Choose. Please update this across all pages.